contracts/ColonyNetworkStorage.sol

```diff
@@ -114,4 +117,9 @@ contract ColonyNetworkStorage is CommonStorage, DSMath {
     require(!_isColony[msg.sender], "colony-caller-must-not-be-colony");
     _;
   }
+
+  modifier calledByMetaColony() {
```

Comment:
Interesting that this didn't exist before

Reply:
We only use this check in `allowedToAddSkill` modifier on the network as only the Meta Colony is allowed to add global skills (stored on the network)